### ISSUE TITLE
Mozilla Firefox Add-ons

### ISSUE THREAD

**Issue by JuanDeDironne** [jeevan-lal/Excel-Data-Filler-V4#26]
Hello

First of all, thank you for your work.

Everything works in Chrome but in Mozilla Firefox, the "add-on" is not available.
Are there plans to put it online again...?

**Comment by jeevan-lal** [comment]
Yes, extension new version already uploaded in Firefox addon but due to the holidays, responses from reviewers may be delayed.

**Comment by JuanDeDironne** [comment]
Thank you for your response. 
Since yesterday the add-on appears under Mozilla Firefox but when you click on to add the extension, the installation seems to be done but the add-on is not integrated into Mozilla.
And if we go to the page "[https://efiller-doc.netlify.app/documentation/#introduction](https://efiller-doc.netlify.app/documentation/#introduction)", in the "_**Download extension**_" section the link to "_**Firefox browser**_" refers to a page indicating that the extension doesn't exist.

**Comment by jeevan-lal** [comment]
Extension firefox version 4.8.3 its auto remove from browser maybe some bugs in this version, We already fixed this issue and upload a new version in firefox addon but due to holiday its not approving pls wait 

![image](https://github.com/user-attachments/assets/b2a30e67-59d8-4baf-be98-82a0daece1be)

**Comment by JuanDeDironne** [comment]
First of all thank you for the work you do.
Then I downloaded the latest version posted online. Version **4.8.6** released online on January 16, 2025.
But when trying to use it I get "**File not found**"
Firefox cannot find the file at "moz-extension://1915f4e9-1917-454a-9969-cfdc7d0c23da/undefined3/firefox/STABLE/production".

**Comment by jeevan-lal** [comment]
This bug fix in version 4.8.7 but its under review..

![Image](https://github.com/user-attachments/assets/c226f1f9-ac00-414f-bcbd-5630841faacd)

**Comment by jeevan-lal** [comment]
Now live  - https://addons.mozilla.org/en-US/firefox/addon/excellent-data-filler/

**Comment by JuanDeDironne** [comment]
Yes, we downloaded it and no more link errors. Thank you for your work :)
